src/main/scala/com/softwaremill/mqperf/mq/Mq.scala

```diff
@@ -12,6 +12,10 @@ trait Mq {
     def send(msgs: List[String])
 
     def close() {}
+
+    def withTimestamp(msg: String): String = msg + "_" + System.currentTimeMillis().toString
```

Comment:
How about cutting off the last N characters from msg, where N is the number of digits in the timestamp?
This way we will preserve the message length described in `msg_size` test parameter.